### ISSUE TITLE
Support Namespaces

### ISSUE THREAD

**Issue by Jay2k1** [inettgmbh/checkmk-proxmox_backup_server#4]
Hi,

you can create namespaces on PBS, so you can easily use one PBS for multiple PVE servers/clusters and separate their backups from each other. If all your backups are organized in namespaces, the current check doesn't see a single backup because it only looks at the root namespace:

```
root@pbs:/# proxmox-backup-client list --repository "$repo" --output-format json-pretty
[]
root@pbs:/# proxmox-backup-client list --repository "$repo" --output-format json-pretty --ns cust1 | head 
[
  {
    "backup-count": 27,
    "backup-id": "155",
    "backup-type": "vm",
    "files": [
      "qemu-server.conf.blob",
      "index.json.blob",
      "client.log.blob",
...
```

You'd have to iterate namespace dependent commands over all available namespaces using the `--ns` switch. You can get a list of all namespaces like so:
```
proxmox-backup-client namespace list --repository "$PBS_REPO:$name" --output-format json-pretty
```

**Comment by marcogabriel** [comment]
Thanks for mentioning this. I assumed we're already checking namespaces. At least we use namespaces already.

@nedjitef is this the latest version?

**Comment by nedjitef** [comment]
No, we don't monitor namespaces in the current check version but we should change that.

In the beginning, when PBS didn't have namespaces, we started using multiple data stores for these purposes.
Since this works quite well, we ourselves didn't need to change that yet. We use them, but not that much.

**Comment by fdriessler** [comment]
Versions 1.0.2 and 1.1.0 support monitoring Namespaces.
It is possible to create seperate services, or monitor all namespaces in a datastore using a single service.

If this feature is still lacking, feel free to open a new Issue.
